src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Indentation/IIndentationService.cs

```diff
@@ -73,7 +47,11 @@ internal static class IIndentationServiceExtensions
         /// </summary>
         public static string GetPreferredIndentation(this SyntaxToken token, Document document, CancellationToken cancellationToken)
         {
+#if CODE_STYLE
+            var sourceText = document.GetTextAsync(cancellationToken).WaitAndGetResult_CanCallOnBackground(cancellationToken);
```

Comment:
I thought the codestyle layer didn't use documents or am I totally off base

Reply:
this is not the *codefixes* layer, which can use documents (but only the public facing side of them).